resources/js/entrypoints/contest-judge.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import Main from 'contest-judge/main';
+import core from 'osu-core-singleton';
+import * as React from 'react';
+import ContestEntryStore from 'stores/contest-entry-store';
+import { parseJson } from 'utils/json';
+
+const store = new ContestEntryStore();
+store.updateWithJson(parseJson('json-entries'));
```

Comment:
This should go in the register callback, top level module code run when the module is first loaded, not when the page is loaded.